addons/website_crm_partner_assign/views/crm_lead_views.xml

```diff
@@ -184,15 +184,11 @@
                 <field name="user_id" position="after">
                     <field name="partner_assigned_id"/>
                 </field>
-                <xpath expr="//span[@t-esc='record.partner_id.value']" position="replace">
-                    <span class="o_text_overflow"
-                        t-esc="record.partner_id.value"
-                        t-att-title="record.partner_id.value"
-                        t-if="record.partner_id.value"/>
+                <xpath expr="//field[@name='partner_id']" position="replace">
```

Comment:
xpath is wrong. It replaces <field name="partner_id"> ouside of the kanban-card

Reply:
Yes, I have made the changes. I think it's working now. Could you please verify?
Thank you.